cdap-ui/app/features/hydrator/controllers/create/toppanel-ctrl.js

```diff
@@ -74,6 +74,11 @@ class TopPanelController{
   }
   saveMetadata() {
     this.ConfigActionsFactory.setMetadataInfo(this.state.metadata.name, this.state.metadata.description);
+    if(this.state.metadata.description.indexOf('/\n/g') === -1) {
```

Comment:
Alright, actually I don't think you need this condition at all. Just do the replace.. If it doesn't find it, it will ignore it anyways.